src/metro/devices/display/fast_plot.py

```diff
@@ -790,8 +795,8 @@ def on_render_completed(self):
 
     def _buildAxisLabel(self, value, x, y, p, flags):
         text = str(value)
-
-        r = p.boundingRect(int(x), int(y), 1, 1, flags, text)
+        
```

Comment:
👀

Reply:
do you have the last diff opened?

<img width="647" alt="grafik" src="https://github.com/user-attachments/assets/ee646e01-434a-4b1e-9296-2d44ded78cc9" />

Reply:
I'm fairly sure I had opened the link anew, but guess I didn't. Cannot find the trailing whitespace now myself anymore, nvm.